designer/client/src/actions/nk/displayProcessCounts.ts

```diff
@@ -15,11 +16,34 @@ export function displayProcessCounts(processCounts: ProcessCounts): DisplayProce
     };
 }
 
+const checkActualCounts = (processCounts: ProcessCounts, processToDisplay: Process) => {
```

Comment:
maybe something like checkPossibleCountsToCalculate will be meaningful? IIUC we handle the situation that, there are changes in the scenario  before the deploy, so the backend calculates old state of the counts, and until someone clicks deploy new version, there will be a difference between frontend and backend.

Reply:
okay, good point